pytorch_lightning/trainer/trainer.py

```diff
@@ -2142,6 +2142,7 @@ def log_dir(self) -> Optional[str]:
 
     @property
     def use_amp(self) -> bool:
+        rank_zero_deprecation("`Trainer.use_amp` is deprecated in v1.6.0 and will be removed in v1.8.0.")
```

Comment:
Include what users should use instead in the deprecation warning so it's easier for them to upgrade.